chart_versions.csv

```diff
@@ -0,0 +1,15 @@
+1.14.000,/charts/rke2-cilium.yaml,true
```

Comment:
CSV is kind of a step backwards in terms of readability and maintainability. I get that it makes it easy to consume from bash and so on, but would you consider using yaml or perhaps just tab/whitespace delimiting instead, so that it's easy for us (the maintainers) to manage?

We regularly bump chart versions and I'd like to make that as easy and mistake-proof as possible.

Reply:
done